pkg/mermaid/mermaidintegrationdiagram.go

```diff
@@ -0,0 +1,88 @@
+package mermaid
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/anz-bank/sysl/pkg/sysl"
+)
+
+type integrationpair struct {
+	firstapp, secondapp string
+}
+
+func GenerateMermaidIntegrationDiagram(m *sysl.Module, appname string) (string, error) {
```

Comment:
Similarly, method signatures will already be called with `mermaid.GenerateMermaidIntegrationDiagram` so could also shorten here to 'GenerateIntegrationDiagram'

Reply:
Addressed!